csharp/PhoneNumbers/PhoneNumbers.csproj

```diff
@@ -29,6 +29,8 @@
   <ItemGroup>
     <EmbeddedResource Include="..\..\resources\*.xml" Exclude="..\..\resources\PhoneNumberMetadataForTesting.xml" />
     <EmbeddedResource Include="..\..\resources\geocoding\**" LinkBase="geocoding" />
+    <EmbeddedResource Include="..\..\resources\timezones\map_data.txt" LinkBase="timezones" />
+    <EmbeddedResource Include="..\..\resources\timezones\windowsZones.xml" LinkBase="timezones" />

```

Comment:
What is this file? I see the comment below but the resources folder is a direct copy of the Google libphonenumber resources [directory ](https://github.com/google/libphonenumber/tree/master/resources) and this would get whipped out.

Reply:
Hi. Okay, I was not aware of the provenance of the directory.  Would it be acceptable to create a new directory that does not exist in the Google libphonenumber? The reason is that it seems more natural for .Net applications to be able to get a TimeZoneInfo object, and the xml file maintained by unicode seems like an easy solution.

However, this part could be omitted if strict conformance to the Google API is required.

Reply:
Per this additional potential functionality I have started work on an idea based on https://github.com/twcclegg/libphonenumber-csharp/issues/160 (libphonenumber-csharp.extensions) it does seem like that could be a good place for this and other OTT functionality. This would certainly add more validity to a side library than just `TryParse`

Reply:
yeah, I think it makes sense to include the dotnet timezone mapping thing in an extension.